code/modules/client/preference/loadout/loadout_accessories.dm

```diff
@@ -4,200 +4,200 @@
 	sort_category = "Accessories"
 
 /datum/gear/accessory/scarf
-	display_name = "scarf"
+	display_name = "Scarf"
```

Comment:
Undo this
![image](https://user-images.githubusercontent.com/25063394/115781950-f7184080-a3b2-11eb-8259-6e02f59e1962.png)

You will break all existing loadouts.